tests/dummy/config/targets.js

```diff
@@ -13,6 +13,7 @@ if (isCI || isProduction) {
   browsers.push('ie 11');
 }
 
-module.exports = {
-  browsers
-};
+module.exports = [
```

Comment:
Temp fix for: https://github.com/emberjs/ember.js/issues/19353

Reply:
We don't need to merge it like this, but I already did the work for this PR so wanted to create it anyways.

Reply:
This is now fixed in `ember-source`.